### PR TITLE
Add napari-lazy-openslide recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@manzt, do you agree with being added as a maintainer?

**Comment by goanpeca** [comment]
Haven't heard from the author. Removing from the maintainers list. We can add them when the feedstock is created if they agree by then.

**Comment by manzt** [comment]
> Haven't heard from the author. Removing from the maintainers list. We can add them when the feedstock is created if they agree by then.

Hi sorry for the delay on this. Happy to be a maintainer.

**Comment by goanpeca** [comment]
No worries @manzt. Thanks :)

**Comment by hmaarrfk** [comment]
azure is down
https://conda-forge.org/status/

File: recipes/napari-lazy-openslide/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - dask-core >=2.20.0
+    - napari-plugin-engine >=0.1.4
+    - numpy >=0.1.19
```
Comment (jaimergp) [inline recipes/napari-lazy-openslide/meta.yaml:25]: Is this correct? I'd assume it means:

```suggestion
    - numpy >=1.19
```

File: recipes/napari-lazy-openslide/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - openslide-python ==1.1.2
```
[the remark this answers is not in the document]
Comment (goanpeca) [reply recipes/napari-lazy-openslide/meta.yaml:26]: Pinging @manzt :)

[the remark this answers is not in the document]
Comment (manzt) [reply recipes/napari-lazy-openslide/meta.yaml:26]: Nope, this is fine to relax as you suggested!

[the remark this answers is not in the document]
Comment (goanpeca) [reply recipes/napari-lazy-openslide/meta.yaml:26]: Thanks for the reply @manzt !

**Comment by jaimergp** [comment]
Windows failures due to `openslide` missing, but it might be available [soon](https://github.com/conda-forge/openslide-feedstock/pull/11).

**Pull request merged**